flutter/macos/sentry_flutter.podspec

```diff
@@ -12,7 +12,7 @@ Sentry SDK for Flutter with support to native through sentry-cocoa.
                          :tag => s.version.to_s }
   s.source_files     = 'Classes/**/*'
   s.public_header_files = 'Classes/**/*.h'
-  s.dependency 'Sentry', '~> 7.0.0'
+  s.dependency 'Sentry', '~> 7.0.3'
```

Comment:
are these 2 files totally equal? ios and macos, can we make a symlink too? or this is actually required because of publishing

Reply:
Hm, that's a good question. I think technically we can make the content work for iOS and macOS with just one file.
Two possible issues I see are
- CocoaPods has some issues with symlinks, that's the reason why I didn't symlink the source folder but all files individually.
- The files are copied into the app project either via Cocoapods or via Flutter itself. I don't know if that works with symlinks.

I guess we'll have to try and see if it works.

Reply:
makes sense, thanks